dfgviewer/typoscript/setup.txt

```diff
@@ -250,6 +257,7 @@ page.2.marks.LANGSWITCHER < lib.langswitcher
 page.2.marks.METADATA < plugin.tx_dlf_metadata
 page.2.marks.URI < plugin.tx_dfgviewer_uri
 page.2.marks.TOOLBOX < plugin.tx_dlf_toolbox
+page.2.marks.AUDIO < plugin.tx_dlf_audioplayer
```

Comment:
Including this line seems to break the DFG viewer: I no longer see the image of the digitized page. When I remove that line, the latest dfg-viewer code works with latest kitodo-presentation (image shown, but without audio support of course).